### PR TITLE
Widen popup to make space for buttons.

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd 

Screen Shot 2019-09-09 at 14 59 21

### CODE REVIEW

File: platformtour/public/css/platformtour.css [hunk 1]
```diff
@@ -138,8 +138,8 @@ tr.introjs-showElement > th {
   position: absolute;
   padding: 10px;
   background-color: white;
-  min-width: 200px;
-  max-width: 300px;
+  min-width: 300px;
```
Comment (stvstnfrd) [inline platformtour/public/css/platformtour.css]: Why do we need to bump the `min-width` and not just `max-width`?

**Review by stvstnfrd — CHANGES_REQUESTED** [review]

File: platformtour/public/css/platformtour.css (hunk 1 above)
Comment (caesar2164) [reply platformtour/public/css/platformtour.css]: `min-width` is actually the necessary one, but I increased `max-width` as well to keep the available range...

Comment (stvstnfrd) [reply platformtour/public/css/platformtour.css]: I'm not sure I follow..

Isn't the range increasing from `100px` (`300 - 200`) to `200px` (`500 - 300`),
while increasing in total possible width by `200px`?

If we're just tweaking the `min-width`, why do we also need to let it get bigger than it used to in other cases?

Comment (caesar2164) [reply platformtour/public/css/platformtour.css]: whoops, I meant for max to `400px`

**Review by stvstnfrd — APPROVED** [review]
+1 to squash and merge.

**Pull request merged**